src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -62,7 +63,8 @@ class MoreInfoMediaPlayer extends LitElement {
                 `
               )}
         </div>
-        ${supportsFeature(stateObj, MediaPlayerEntityFeature.BROWSE_MEDIA)
+        ${!isUnavailableState(stateObj) &&
```

Comment:
```suggestion
        ${!isUnavailableState(stateObj.state) &&
```
Ensure to pass the state and not the stateObj.

Reply:
Duh...I shouldn't push changes that late at night 🙈